appservice/src/deploy/delayFirstWebAppDeploy.ts

```diff
@@ -16,6 +16,8 @@ export async function delayFirstWebAppDeploy(client: SiteClient, asp: WebSiteMan
             if (client.isFunctionApp) {
                 resolve();
             }
+
+            const asp: WebSiteManagementModels.AppServicePlan | undefined = await aspPromise;
```

Comment:
Shouldn't we still be using this logic here?

```
try {
   asp = await aspPromise;
} catch {
   asp = undefined;
}
```

Reply:
It's already in a bigger try/catch